src/PostalCodes/IsoCountryCodeConverter.cs

```diff
@@ -0,0 +1,37 @@
+﻿using System;
+using System.Linq;
+
+namespace PostalCodes
+{
+    /// <summary>
+    /// Class IsoToVistaprintCountryCodeConverter.
```

Comment:
Fix comment (though this comment really doesn't add anything, so perhaps we should just remove it.)